Nagstamon/Servers/IcingaDBWeb.py

```diff
@@ -220,7 +220,8 @@ def _get_status(self):
                         self.new_hosts[host_name].notifications_disabled = not int(h.get('notifications_enabled') or '0')
                         self.new_hosts[host_name].flapping = bool(int(h['state']['is_flapping'] or 0))
                         #s['state']['is_acknowledged'] can be null, 0, 1, or 'sticky'
-                        self.new_hosts[host_name].acknowledged = bool(int(h['state']['is_acknowledged'].replace('sticky', '1') or 0))
+                        #self.new_hosts[host_name].acknowledged = bool(int(h['state']['is_acknowledged'].replace('sticky', '1') or 0))
+                        self.new_hosts[host_name].acknowledged = h['state']['is_acknowledged'] if isinstance(h['state']['is_acknowledged'], (bool)) else bool(int(h['state']['is_acknowledged'].replace('sticky', '1') or 0))
```

Comment:
Shouldn't it work with `bool` instead of `(bool)` too?

Reply:
works for me with the correction from @HenriWahl